src/Orc.CsvTextEditor/Services/CsvTextEditorInstance.cs

```diff
@@ -493,6 +496,18 @@ private void OnTextChanged(object sender, EventArgs eventArgs)
             UpdateTextChangingIterator();
 
             RaiseTextChanged();
+
+            if (_elementGenerator.UnfreezeColumnResizing())
+            {
+                if (_refreshViewTimer != null)
+                {
+                    _refreshViewTimer.Stop();
+                }
+                _refreshViewTimer = new DispatcherTimer();
```

Comment:
Not sure about re-creating a new timer every time. Can't we just re-use the same instance?

Reply:
And when reading this code, it looks like there will always be a timer running in the end, please create the timer in the ctor and start in OnLoaded and stop in OnUnloaded.